src/builder.cpp

```diff
@@ -501,14 +509,15 @@ MeshInstance3D* Builder::build_entity_mesh(int idx, LMEntity& ent, Node3D* paren
 	// Create collisions if needed
 	switch (coltype) {
 	case ColliderType::Mesh:
-		add_collider_from_mesh(parent, mesh, colshape);
+		add_collider_from_mesh(parent, collision_mesh, colshape);
 		break;
 
 	case ColliderType::Static:
-		switch (colshape) {
-		case ColliderShape::Convex: mesh_instance->create_multiple_convex_collisions(); break;
-		case ColliderShape::Concave: mesh_instance->create_trimesh_collision(); break;
-		}
+		StaticBody3D* static_body = memnew(StaticBody3D());
+		static_body->set_name(String(mesh_instance->get_name()) + "_col");
+		parent->add_child(static_body, true);
+		static_body->set_owner(m_loader->get_owner());
+		add_collider_from_mesh(static_body, collision_mesh, colshape);
```

Comment:
Does this change the behavior from `mesh_instance->create_multiple_convex_collisions()` or `mesh_instance->create_trimesh_collision()`?